wheels/bx-python/meta.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+name: bx-python
+version: 0.8.2
+type: wheel
+purepy: false
+prebuild:
+  wheel: pip install --index-url https://wheels.galaxyproject.org/simple/ --extra-index-url https://pypi.python.org/simple numpy==1.15.1 python-lzo
```

Comment:
@natefoo I modified this line from the original ```"for py in /python/cp*-`uname -m`; do ${py}/bin/pip install --index-url https://wheels.galaxyproject.org/simple/ numpy python-lzo || exit 1; done"``` in https://github.com/galaxyproject/starforge/blob/master/wheels/build/wheels.yml#L12 , but it fails on Linux builds because it cannot find pip. What's the correct syntax?

Reply:
And should we add Cython?

Reply:
Hmm, since we no longer set up `/python`, this is an issue. But expecting `/python` was a hack in the first place, Starforge knows where the build pythons are and should do this for you automatically if you specify a package to install with pip. Let me see if I can get that working quickly.

Reply:
So I can just remove the `prebuild:` and see if it works with your latest starforge commit?